connector_ebisumart/components/ebisumart_adapter.py

```diff
@@ -71,6 +73,35 @@ def _fetch_data(self, endpoint, params=None):
         if response.status_code == 200:
             return response.json()
 
+    def _get_filters(self, last_fetch_date):
```

Comment:
To make the scope of the method a bit clearer.
```suggestion
    def _get_date_filters(self, last_fetch_date):
```